docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -121,7 +166,9 @@ root@host $ systemctl start docker.stash-origin
     You must [register](install-origin.md#registering-the-origin) the origin before considering it a
     production service.
 
+### Running on origin kubenetes with systemd
 
+TODO
```

Comment:
I don't think we need to give systemd-related Kubernetes instructions – once the deployment is in k8s, users shouldn't have to worry about maintaining a persistent service via systemd!

Reply:
Right, that's within the container using supervisord. Systemd would only really be used on bare metal to manage services, which is unnecessary with k8s

Reply:
> Sorry, this topic should be supervisord

I don't think this is necessary: the container should start supervisord and thus the relevant services. The operator should ideally not have to interact with supervisord at all!